ui/src/app/core/vertical-action-bar.component.ts

```diff
@@ -15,15 +15,11 @@ export class VerticalActionBarComponent {
   ) {
   }
 
-  get visibleActions(): TableActionDefinition[] {
-    return this.actions.filter(it => it.isVisible)
-  }
-
   get primaryActions(): TableActionDefinition[] {
-    return this.visibleActions.filter(it => it.primary)
+    return this.actions.filter(it => it.primary)
```

Comment:
By removing the isVisible filter, does that mean that these actions can now be performed on *any* button?  I don't remember how this works.

Reply:
We've replace the visible flag with disable property in the template. Previously, the buttons would disappear. It is now disabled.

Reply:
Makes sense now.  Thanks.